### DISCUSSION TITLE
Libraries already working?

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#214] [Q&A] [answered]
I was trying to set up a global library using the 'Malt Library Path' addon preference. Should that already work? I know its not labled **Done** yet but the file path preference already exists so I was wondering. I used the exact example directory layout if that helps but the test shader function that I stored in the library does not show up.

**Comment by pragma37** [comment, marked as answer]
Long story short, not yet.
That option is actually much older and it worked at some point IIRC.

I plan to work on Malt libraries soon, however.
